README.md

```diff
@@ -102,7 +118,7 @@ If you're using Visual Studio Code for development, it's recommended that you:
 
 8. Open another terminal window and in this new window run the command `npm run tunnel-mac-linux` (if you are on a Mac or Linux) or `npm run tunnel-windows` (if you are on Windows). This 'tunnels' your local API server: makes it available externally so your app running in Expo Go can access it. You should see a message saying your `your url is: https://.............` -- this is the URL of your local API server, make a note of it as you'll need it in a minute.
 
-   > This URL should include the value you set for `API_TUNNEL_SUBDOMAIN` in your `api/.env` file, e.g. something like `https://nadia-bloggs-abc123def456ghi789.loca.lt`
+   > This URL should include the value you set for `API_TUNNEL_SUBDOMAIN` in your `api/.env` file, e.g. something like `https://nadia-bloggs-abc123def456ghi789.loca.lt`. If you do not see this URL, then double-check your `API_TUNNEL_SUBDOMAIN` as discussed in step 5. Ensure it is not prefixed with `https://`.
```

Comment:
Maybe worth making this even clearer here? `API_TUNNEL_SUBDOMAIN` should only include the part after `https://` and before `.local.lt`?

Reply:
I have attempted to make this clearer by clarifying the omission of `https://` for API_TUNNEL_SUBDOMAIN but not STA_BASE_URL and omitting .local.lt from API_TUNNEL_SUBDOMAIN